requirements.txt

```diff
@@ -1,14 +1,18 @@
-Jinja2==2.7.3
-MarkupSafe==0.23
+flake8==2.5.4
 gunicorn==19.3.0
 invoke==0.10.1
+Jinja2==2.7.3
+MarkupSafe==0.23
+mccabe==0.4.0
 peewee==2.6.1
+pep8==1.7.0
 psycopg2==2.6
--e git+https://github.com/loisaidasam/pyslack@8a6397b7911e8e82d0fbbccc7ae0c6190b32f2f4#egg=pyslack-master
+pyflakes==1.0.0
```

Comment:
Perhaps it would make sense to move these development dependencies to `dev_requirements.txt` or such so they are not installed into production environments. What do you think @k0nserv?